src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IKeyValueAdapter.cs

```diff
@@ -15,7 +15,9 @@ internal interface IKeyValueAdapter
 
         bool CanProcess(ConfigurationSetting setting);
 
-        void InvalidateCache(ConfigurationSetting setting = null);
+        void OnConfigurationRefresh(ConfigurationSetting setting = null);
+
+        void OnConfigurationUpdated();
```

Comment:
How about `OnChangeDetected` and `OnConfigUpdated`?

Reply:
Sure, I think that's clearer for the first one